tests/test_models.py

```diff
@@ -322,6 +322,27 @@ def test_make_guidewindow():
     assert guidewindow_model.validate() is None
 
 
+# Testing all reference file schemas
+def test_reference_file_model_base(tmp_path):
+    # Set temporary asdf file
+
+    # Get all reference file classes
+    tags = [t for t in stnode.NODE_EXTENSIONS[0].tags if "/reference_files/" in t.tag_uri]
+    for tag in tags:
+        # skycells doesn't use EXPECTED_COMMON_REFERENCE
+        if repr(tag).find("skycells") > 0:
+            return
+        schema = asdf.schema.load_schema(tag.schema_uris[0])
+        # Check that schema references common reference schema
+        allofs = schema["properties"]["meta"]["allOf"]
+        found_common = False
+        for item in allofs:
+            if item == EXPECTED_COMMON_REFERENCE:
+                found_common = True
+        if not found_common:
+            raise ValueError("Reference schema does not include ref_common")  # pragma: no cover
+
+
```

Comment:
```suggestion
```

There is already a test in rad that checks that reference file schemas reference ref_common https://github.com/spacetelescope/rad/blob/244328cc39530e67a3d3763442a49e95f22dc163/tests/test_schemas.py#L303
I vote for not adding another test here.

Reply:
Vote wins the election.